src/style.css

```diff
@@ -1 +1,16 @@
-/* styles go here */
+.personal_information_label,
+.registration_fieldset_email,
+.interesting_facts_about_you,
+.additional_info,
+.asking_about_cats_div {
```

Comment:
you can just add the same class `form-field` to each `label` and style it there.